CHANGELOG.md

```diff
@@ -3,6 +3,153 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.0.0-alpha.15
+
+_Jan 13, 2023_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+TODO INSERT HIGHLIGHTS
```

Comment:
Some help with the highlights would be greatly appreciated :D

Reply:
The introduction of the `slots` and `slotsProps` on the pickers is probably worth highlighting

Reply:
Spannish and Belarusian locale

Reply:
Updated